### ISSUE TITLE
No module named 'typing_extensions' during test/algorithms

### ISSUE THREAD

**Issue by paulcrouther** [alik-git/mbrl-lib#1]
## Steps to reproduce

  1. Follow the readme guide to install and activate the RLDucky virtualenv
  2. Follow the readme and clone the duckietown-mbrl repo:
  3. Run `python -m pytest tests/algorithms`

## Observed Results

  * What happened?  This could be a description, log output, etc.
  * Error resulted:  ```File "/Users/paul/opt/anaconda3/envs/RLDucky/lib/python3.8/site-packages/hydra/test_utils/test_utils.py", line 20, in <module>
    from typing_extensions import Protocol
ModuleNotFoundError: No module named 'typing_extensions' ```

## Expected Results

  * What did you expect to happen?
  * Expected no issue with running the test.

## Relevant Code

  ```
git clone https://github.com/alik-git/duckietown-mbrl-lib
cd duckietown-mbrl-lib
conda activate RLDucky
pip install -e ".[ducky]"`

python -m pytest tests/algorithms

  ```

Managed to suppress this according to: https://stackoverflow.com/questions/69108343/modulenotfounderror-no-module-named-typing-extensions using the following:

  ```
pip install typing-extensions
  ```

Which updates to most recent versions. Probably an issue with MBRL and not this repo directly.

**Comment by alik-git** [comment]
Yeah so this should be fixed in [this commit](https://github.com/alik-git/duckietown-mbrl-lib/commit/46f9571634cad22c58182c5b4cc9e35fe40b9e02) but I'm not sure since pytorch is also included in those requirements, but still isn't installed for some reason. Weird.
